src/http/RegisteredHeaders.h

```diff
@@ -57,7 +57,7 @@ enum HdrType {
     FORWARDED,                      /**< RFC 7239 */
     FROM,                           /**< RFC 7231 */
     HOST,                           /**< RFC 7230 */
-    HTTP2_SETTINGS,                 /**< RFC 7540 */
+    /*HTTP2_SETTINGS,*/             /* obsolete RFC 7540 */
```

Comment:
I do not understand why we should keep a collection of obsolete headers that we do not (and probably will never) use in the code.

```suggestion
```

Yes, I know there are a few other similar commented out entries. If this specific change is done for consistency sake, would you be willing to remove all such entries in this or a dedicated followup PR?

Reply:
I am not changing how the header registrations are handled in this PR. It is scoped purely at RFC9113 differences from RFC 7540.